elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/filter/CriterionFilterOperation.java

```diff
@@ -34,6 +34,12 @@
 
     private static String ALIAS_DELIM = "__";
 
+    private static String SPECIAL_CHARACTER = "%";
+
+    private static String MATCHALL_CHARACTER = SPECIAL_CHARACTER;
+
+    private static String ESCAPE_CHARACTER = "\\";
+
```

Comment:
I know codacy probably complained about these, but it's just a tool and I don't think that having these constants adds to the readability of the code. In fact since we set `MATCHALL_CHARACTER = SPECIAL_CHARACTER` we're asking for unintentional behavior changes if we changed `SPECIAL_CHARACTER`